the-name-card/src/components/CardHolder/CardHolder.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC, PropsWithChildren, ReactNode } from 'react';
+import { Section } from '..';
+import './CardHolder.scss';
+
+interface CardHolderProps {
+	children: ReactNode | ReactNode[];
+}
+
+const CardHolder: FC<PropsWithChildren<CardHolderProps>> = ({ children }) => {
```

Comment:
don't have to use `PropsWithChildren` wrapper if you are already defining the `children` attribute in the props

also good to stick to one convention for the entire project